packages/babel-plugin-proposal-private-methods/test/fixtures/accessors-loose/options.json

```diff
@@ -0,0 +1,15 @@
+{
+  "plugins": [
+    [
+      "external-helpers",
+      {
+        "helperVersion": "7.1000.0"
+      }
+    ],
+    ["proposal-private-methods", { "loose": true }],
+    ["proposal-class-properties", { "loose": true }],
+    "transform-classes",
+    "transform-block-scoping",
+    "syntax-class-properties"
```

Comment:
Not specific to this test/PR but was wondering if we actually need to include these plugins?

`"transform-block-scoping", "syntax-class-properties"`